mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -68,17 +83,20 @@
                   Types.TimestampType.withoutZone(), Types.StringType.get(), Types.BinaryType.get(),
                   Types.DecimalType.of(3, 1));
 
-  @Parameters(name = "fileFormat={0}, engine={1}, catalog={2}")
+  @Parameters(name = "fileFormat={0}, engine={1}, catalog={2}, cboEnable={3}")
   public static Collection<Object[]> parameters() {
     Collection<Object[]> testParams = new ArrayList<>();
     String javaVersion = System.getProperty("java.specification.version");
+    List<Boolean> cboEnables = ImmutableList.of(true, false);
```

Comment:
Instead of running all tests for both CBO on/off, can we turn it on just in those one or two unit test cases where we want to test it, by setting it via `shell.setHiveSessionValue()` at the beginning of the test? We're trying to prevent the number of test runs from exploding via the combination of ever-increasing test parameters.

Reply:
@marton-bod OK, I have moved CBO test to some join test cases.